docs/FAQs/UseASubscriptionInAJsEvent.md

```diff
@@ -0,0 +1,53 @@
+### Question
+
+How do I access the value of a subscription from within a Reagent or JavaScript event (such as on-click)?
+
+### The Wrong Way
+
+You should NOT do this:
+```clj
+[:input {:type "button" 
+         :value "Click me!"
+         :on-click #(let [sub-val  @(subscribe [:something])]   ;;<--- Eeek
+       ....)}]
+```
+
+because that `subscribe`:
+1. might create a memory leak (the subscription might not be "freed")
+2. makes the event handler impure (it grabs a global value)
```

Comment:
I would remove this point.
Such event handlers are almost always anonymous functions created and used right then and there. And if you put `@` outside in `let`, it won't make the value "non-global".
I think creating a closure over a locally referenced value is just fine in this particular case, no matter where the data came from.